### PR TITLE
Adding release notes to the documentation.

### PR DESCRIPTION
**Pull request by watfagithub**

### CODE REVIEW

**Comment by watfagithub** [comment]
image

image

**Comment by jkusa** [comment]
the release will be `1.0.0-beta.1`

File: pages/guide/09-start.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+layout: guide
+group: guide
+title: Release Notes
+toc: true
+---
+
+# Release [1.0.0-beta.1](https://github.com/yavin-dev/app/releases)
```
Comment (jkusa) [inline pages/guide/09-start.md]:
```suggestion
# Release [1.0.0-beta.1](https://github.com/yavin-dev/app/releases/tag/v1.0.0-beta.1)
```

File: pages/guide/09-start.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+# How to upgrade to a new release
+
+- You can upgrade your Yavin app version with the following commands :
+  > ```shell
+  > cd app
+  > git pull origin
+  > ```
+  >
+  > Make sure you on the right version by running  ```yavin -version```
```
Comment (jkusa) [inline pages/guide/09-start.md]:
```suggestion
```

File: pages/guide/09-start.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+[yahoo/elide#1876](https://github.com/yahoo/elide/pull/1876) N+1 performance improvements in the JPA data store.  The JPA store will now return proxied collections (allowing the ORM to batch fetch the collection) and filter, sort, and paginate in memory whenever fetching a collection (N>1) of collections.  There is a feature flag to enable/disable this behavior.
+
+
+### <img src="/assets/images/Release_api_icon.png" alt="API Changes logo" style="height: 40px; margin-bottom: -8px"> API Changes
+
+[yahoo/elide#1897](https://github.com/yahoo/elide/pull/1897) The Aggregation Store now supports filters on metrics that have not been requested/projected in the client request. (Note: This is not yet available in Yavin UI)
+
+> For a complete list of API changes, check out the latest [Elide Changelog](https://github.com/yahoo/elide/blob/7a11ee300605ed0130d89a036ed221d49f1b1d9c/changelog.md#500-pr32)
+{:.info}
```
Comment (kevinhinterlong) [inline pages/guide/09-start.md]: This can probably go under `Updates & Additions` since it's not available in the UI yet

**Review by jkusa — DISMISSED** [review]

File: pages/guide/09-start.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+
+### <img src="/assets/images/Release_api_icon.png" alt="API Changes logo" style="height: 40px; margin-bottom: -8px"> API Changes
+
+> For a complete list of API changes, check out the latest [Elide Changelog](https://github.com/yahoo/elide/blob/7a11ee300605ed0130d89a036ed221d49f1b1d9c/changelog.md#500-pr32)
+{:.info}
+
+### <img src="/assets/images/Release_updates_icon.png" alt="updates logo" style="height: 40px; margin-bottom: -8px"> Updates & Additions
+
+[yahoo/elide#1897](https://github.com/yahoo/elide/pull/1897) The Aggregation Store now supports filters on metrics that have not been requested/projected in the client request. (***Note***: This is not yet available in Yavin UI)
```
Comment (kevinhinterlong) [inline pages/guide/09-start.md]:
```suggestion
- [yahoo/elide#1897](https://github.com/yahoo/elide/pull/1897) The Aggregation Store now supports filters on metrics that have not been requested/projected in the client request. (***Note***: This is not yet available in Yavin UI)
```

**Review by kevinhinterlong — APPROVED** [review]

**Pull request merged**